site/views/draft/popup-game.njk

```diff
@@ -66,108 +60,108 @@ aside:
   }
   .popup-content {
     flex: 1;
     margin: auto;
   }
+
+  .popup-content[collapsed] .popup-g-hide-btn,
+  .popup-content:not([collapsed]) .popup-g-continue-btn {
+    display: none;
+  }
+  
 </style>
+
 <p>This cool game will let you play with a popup.</p>
 <p>Drag a trigger element and watch how the popup and its arrow move. Use the mouse wheel to resize the container. Use shift to switch the resize direction.</p>
 <p>The green area on the edges of the container is determined by the offsetContainer value and means the safe zone in which popup can't come. The red zone on the edges of popup demonstrates the limits of arrow placement.</p>
-<h2>Horizontal, direction: LTR</h2>
-<section class="row">
-  <div>
-    <div class="boundary-container boundary-container-1">
-      <esl-d-popup-game trigger="::find(.game-popup-trigger)">
-        <esl-trigger
-          target="#game-popup-h-ltr"
-          mode="toggle"
-          class="game-popup-trigger"
-          style="top: 124px; left: 124px;">
-          🚀
-        </esl-trigger>
-      </esl-d-popup-game>
-    </div>
-  </div>
-  <esl-popup
-    id="game-popup-h-ltr"
-    class="popup-game popup-h"
-    position="top"
-    container=".boundary-container-1"
-    dir="ltr"
-    margin-arrow="35">
-    <div class="popup-content-wrapper">
-      <div class="popup-content"> 🇱 🇹 🇷 🐶 🐼</div>
-    </div>
-    <span class="esl-popup-arrow"></span>
-  </esl-popup>
-</section>
-<h2>Horizontal, direction: RTL</h2>
 <section class="row">
-  <div>
-    <div class="boundary-container boundary-container-2">
-      <esl-d-popup-game trigger="::find(.game-popup-trigger)">
-        <esl-trigger
-          target="#game-popup-h-rtl"
-          mode="toggle"
-          class="game-popup-trigger"
-          style="top: 124px; left: 124px;">
-          🚀
-        </esl-trigger>
-      </esl-d-popup-game>
-    </div>
-  </div>
-  <esl-popup
-    id="game-popup-h-rtl"
-    class="popup-game popup-h"
-    position="top"
-    container=".boundary-container-2"
-    dir="rtl"
-    margin-arrow="35">
-    <div class="popup-content-wrapper">
-      <div class="popup-content"> 🇷 🇹 🇱 🦝 🐱 </div>
-    </div>
-    <span class="esl-popup-arrow"></span>
-  </esl-popup>
-</section>
-<h2>Vertical</h2>
-<section class="row">
-  <div>
-    <div class="boundary-container boundary-container-3">
-      <esl-d-popup-game trigger="::find(.game-popup-trigger)">
-        <esl-trigger
-          target="#game-popup-v"
-          mode="toggle"
-          class="game-popup-trigger"
-          style="top: 124px; left: 124px;">
-          🚀
-        </esl-trigger>
-      </esl-d-popup-game>
-    </div>
+  <div class="col-12">
+    <uip-root>
+      <script type="text/html"
+              label="Horizontal"
+              uip-snippet
+              uip-snippet-js="js-snippet-popup-game">
+        <esl-d-popup-game trigger="::find(.game-popup-trigger)">
+          <esl-trigger
+            target="#game-popup"
+            mode="toggle"
```

Comment:
```suggestion
            track-click="not" 
            track-hover
            hover-hide-delay="250"
```

`toggle` is a default
the hover behavior looks more suitable for demo purpose here